Source/Tools/relive_api/relive_api_test.cpp

```diff
@@ -81,37 +81,56 @@ template <typename... Ts>
     return concat(kAODir, fileName);
 }
 
+[[nodiscard]] std::vector<u8>& getStaticFileBuffer()
+{
+    static std::vector<u8> result;
+    return result;
+}
+
+[[nodiscard]] std::vector<u8>& readFileIntoStaticFileBuffer(const std::string& filePath)
```

Comment:
does this really make a huge change to performance? These static buffers make adding threading later harder can sometimes cause issues since the buffer can contain unexpected data

Reply:
It contributed to the speedup I have seen. We can change `static` to `thread_local` if we add threading in the future.